python/mlc_chat/cli/compile.py

```diff
@@ -152,6 +168,8 @@ def _check_prefix_symbols(prefix: str) -> str:
         prefix_symbols=parsed.prefix_symbols,
         output=parsed.output,
         max_sequence_length=parsed.max_sequence_length,
+        sliding_window=parsed.sliding_window,
+        sliding_window_chunk_size=parsed.sliding_window_chunk_size,
```

Comment:
Given that `chunk_size` does not affect compilation outcome, let's remove it from compilation